eslint-plugin-defaultvalue/lib/rules/default-value.ts

```diff
@@ -194,8 +194,16 @@ const extractCallExpression = (
         // This will never have a defaultValue.
         return undefined;
       }
+      // Should not be documented.
+      if (readonly) {
+        return undefined;
+      }
```

Comment:
Please explain why you did this change and add it into the commit message. I do not really understand the reasoning.

Reply:
@timowolf It says in the commit message: exclude readonly member call expressions

It was previously discussed to remove readonly expressions (and also done), this is just covering an edge case